src/node/routes/login.ts

```diff
@@ -28,13 +29,15 @@ export class RateLimiter {
 
 const getRoot = async (req: Request, error?: Error): Promise<string> => {
   const content = await fs.readFile(path.join(rootPath, "src/browser/pages/login.html"), "utf8")
+  const lng = req.args["lng"] || "en"
+  i18n.changeLanguage(lng)
   const appName = req.args["app-name"] || "code-server"
-  const welcomeText = req.args["welcome-text"] || `Welcome to ${appName}`
-  let passwordMsg = `Check the config file at ${humanPath(os.homedir(), req.args.config)} for the password.`
+  const welcomeText = req.args["welcome-text"] || (i18n.t("WELCOME", { app: appName }) as string)
```

Comment:
Is this `as string` necessary? I would think `t` returns a `string`

Reply:
We can always look in a followup PR. I don't want to hold you back so it's fine as is for now. Thank you for trying that!